ios-target/ViewModel/Home/TopicsViewModel.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  TopicsViewModel.swift
+//  ios-target
+//
+//  Created by sol manini on 7/2/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import Foundation
+
+enum TopicsViewModelState: Equatable {
+  case loading
+  case error(String)
+  case idle
+}
+
+protocol TopicViewModelDelegate: class {
+  func didUpdateState()
+  func topicSelected(topic: Topic)
+}
+
+class TopicsViewModel {
+  var topics: [Topic] = []
+  var cellHeight = 50
+  
+  var topicsCount: Int {
+    return topics.count
+  }
+  
+  var collectionHeight: Int {
```

Comment:
this one is also for display purposes right? if that's the case, you could put it on the view.